### PR TITLE
Update Theia editor and Exec plugin

### PR DESCRIPTION
**Pull request by garagatyi**
### What does this PR do?
Use direct links to che-plugin.yaml instead of tar.gz archives
to simplify release of new version.

### CODE REVIEW

File: plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ name: theia-ide
 title: Eclipse Theia for Eclipse Che
 description: Eclipse Theia
 icon: https://raw.githubusercontent.com/theia-ide/theia/master/logo/theia-logo-no-text-black.svg?sanitize=true
-url: https://github.com/ws-skeleton/che-editor-theia/releases/download/untagged-66e654fb4d0739982113/che-editor-plugin.tar.gz
+url: https://raw.githubusercontent.com/ws-skeleton/che-editor-theia/f4b077eb01cd6373b0343f6029e5824348238a08/etc/che-plugin.yaml
```
Comment (benoitf) [inline plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: https://github.com/ws-skeleton/che-editor-theia/releases/download/untagged-cf91340b0b4af08ec756/che-editor-plugin.tar.gz ?

I don't like that hack around direct yaml link

Comment (garagatyi) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: In our case tar.gz just contains a single yaml. Does it make sense to package it as an archive?

Comment (benoitf) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: it's automatically published

Comment (garagatyi) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: Setting up automatic publishing in Eclipse repos is way trickier

Comment (benoitf) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: it's travis so AFAIK we have enabled travis on a lot of che repositories ?

Comment (garagatyi) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: Yes, but still requires the creation of issues in the bugzilla

Comment (garagatyi) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: But I'm OK to switch to archive if it is important for you

Comment (benoitf) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: would be great, thx

Comment (garagatyi) [reply plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml]: Updated the PR

**Pull request merged**